app/controllers/internal/broadcasts_controller.rb

```diff
@@ -56,7 +56,7 @@ def destroy
   private
 
   def broadcast_params
-    params.permit(:title, :processed_html, :type_of, :banner_style, :active)
+    params.permit(:title, :processed_html, :type_of, :banner_style, :active, :last_active_at)
```

Comment:
Hm, looking at this again, do we actually need `last_active_at` here? We don't allow the user to update this data, we update it internally, so it shouldn't actually be necessary, right?

Reply:
This is a holdover from my previous work. 🤦‍♀️